MIGRATION.md

```diff
@@ -1,16 +0,0 @@
-#Migrate AWS RDS databases to use different private subnets
```

Comment:
Why we moved this from Migration to Maintenance file?

Reply:
moved it to maintenance because migration is a maintenance task, and we have new task for CA update, so in order to avoid creating a new file every time we have a new maintenance task, this changes has been introduced

Reply:
Hmm, I thought that we need to have like from version to version how to Migrate and not like delete this every time we update version, because like this we don't have any track of migration steps from version to version. We can check this at today meeting.